AutoNode/__init__.py

```diff
@@ -0,0 +1,86 @@
+import os
+import sys
+import json
+import warnings
+
+from pyhmy import (
+    cli,
+    Typgpy
+)
+
+from .common import (
+    harmony_dir,
+    node_dir,
+    node_sh_log_dir,
+    bls_key_dir,
+    saved_validator_path,
+    imported_bls_pass_file_dir,
+    imported_wallet_pass_file_dir,
+    cli_bin_path,
+    saved_node_path,
+    validator_config,
+    node_config
+)
+
+
+if sys.version_info.major < 3:
+    warnings.simplefilter("always", DeprecationWarning)
+    warnings.warn(
+        DeprecationWarning(
+            "`AutoNode` does not support Python 2. Please use Python 3."
+        )
+    )
+    warnings.resetwarnings()
```

Comment:
why we don't exit if we don't have python 3?

Reply:
added exit